drivers.py

```diff
@@ -92,11 +102,151 @@ def upload_file(self, src, obj_path):
         return dest
 
 
+class GoogleDriveDriver(Driver):
+    """Driver to handle connection to Google Drive"""
+
+    def __init__(self, config):
+        super(GoogleDriveDriver, self).__init__(config)
+
+        try:
+            self._credentials = service_account.Credentials.from_service_account_file(
+                Path(config.google_drive.service_account_file),
+                scopes=["https://www.googleapis.com/auth/drive.file"],
+            )
+
+            self._service: Resource = build(
+                "drive", "v3", credentials=self._credentials
+            )
+
+            self._folder = config.google_drive.folder
+            folder_exists, self._folder_id = self._folder_exists(self._folder)
+
+            if not folder_exists:
+                self._folder_id = self._create_folder(self._folder)
+
+            for email in get_share_with(config.google_drive):
+                if email:
```

Comment:
it would be better if get_share_with returned only list of valid emails or empty list

Reply:
done